params/astria_config.go

```diff
@@ -146,6 +169,42 @@ func NewAstriaForks(forks map[string]AstriaForkConfig) (*AstriaForks, error) {
 			orderedForks[i].SnapshotChecksum = currentFork.SnapshotChecksum
 		}
 
+		if currentFork.AppSpecificOrdering != nil {
+			if len(currentFork.AppSpecificOrdering) == 0 {
+				log.Debug("currentFork.AppSpecificOrdering before", "ordering", fmt.Sprintf("%v", currentFork.AppSpecificOrdering))
+			} else {
+				log.Debug("currentFork.AppSpecificOrdering before", "ordering", fmt.Sprintf("%v", currentFork.AppSpecificOrdering))
+			}
```

Comment:
```suggestion
			log.Debug("currentFork.AppSpecificOrdering before", "ordering", fmt.Sprintf("%v", currentFork.AppSpecificOrdering))
```

the logs are the same